### PR TITLE
Project

### PR DESCRIPTION
**Pull request by davidobi**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.md
```
Comment (elmar8287) [inline .gitignore]: - [x] Please, consider adding exactly the `test.md` file into the `.gitignore` file, as required in the project, to ignore it while you will pushing.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <link rel="stylesheet" href="style.css">
+
+    <title>My First Microverse Push to Git</title>
+
+</head>
+<body>
+        <h1>Hello Microverse</h1>
```
Comment (elmar8287) [inline index.html]: - [x] Please, try to add a class name for the `h1` tag and make style, instead of directly styling the HTML tag.

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @davidobi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [x]  Check the comments under the review.
- [x] Please, fix the title and description for Pull Request - it should tell about what you are going to do in this milestone
- [x] Please, consider that your branch name should be descriptive (e.g.: refactor-authentication, user-content-cache-key, make-retina-avatars), so that others can see what is being worked on.
- [x] Please, consider modifying the README file and adapting it to your project: 
- write a description
- delete a demo link (it is not necessary for this project)
- delete empty sections
- write how to set up your project (hint: use `git clone` command)
- Keep only one author, as you are alone in this project
- Fix your social network links
- Modify the link to the MIT.md file and also create it
- Modify Acknowledgments

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by davidobi** [comment]
@elmar8287  Thanks so much for your suggestions, I have implemented all the changes. Please review the code in: https://github.com/davidobi/Hello_Microverse/pull/5

**Pull request closed without merging**